### DISCUSSION TITLE
Why nginx-ingress-controller?

### DISCUSSION THREAD

**Discussion by zkryakgul** [kubernetes/dashboard#8509] [General]
I wonder something about the new v3.0.0-alpha0 version. As this project is under the Kubernetes organization why you choose [nginx-ingress-controller](https://docs.nginx.com/nginx-ingress-controller/installation/) project as a ingress controller instead of another project under the kubernetes community [kubernetes/ingress-nginx](https://github.com/kubernetes/ingress-nginx)

**Comment by floreks** [comment]
No particular reason behind this. It was just something I was familiar enough with to easily set up. You are free to disable it though and use any custom solution.

**Reply by ViliusS** [reply]
I have more interesting question. Install documentation and v3.0.0-alpha0 release notes links to nginx-ingress-controller but 7.0.0 Helm Chart actually [installs Ingress NGINX Controller for Kubernetes](https://github.com/kubernetes/dashboard/blob/a261b9b5db3b0135bf2ffef00a78682dd2eda0f2/charts/kubernetes-dashboard/Chart.yaml#L33-L36) . So which is actually needed?

**Reply by floreks** [reply]
It was true for alpha version. Official v3 release is now using nginx controller from kubernetes org as a dependency, as suggested by @zkryakgul. Also, as mentioned in the release notes

> cert-manager, nginx and metrics-server are now disabled by default. Only kong dependency is required.

**Reply by ViliusS** [reply]
I have made a PR https://github.com/kubernetes/dashboard/pull/8764 to make it more clear for others.
